packages/server/src/services/carpool.service.ts

```diff
@@ -143,6 +143,14 @@ export class CarpoolService {
         return drivers.map(driver => mapCarpoolToDto(driver.carpool));
     }
 
+    /**
+     * Finds the carpool that the givern driver is driving in
+     * @param driverId - ID of the driver who is in the carpool
+     */
+    public async findCarpoolIdByDriverId(driverId: string): Promise<string> {
+        const driver = await this._driverRepository.findOne(driverId);
```

Comment:
Probably want to either use `findOneOrFail` or check for undefined before returning the carpool ID, e.g. `return driver?.carpoolId` <-- will want to update the return type to include undefined in this case tho.